drivers/base/bootvid/precomp.h

```diff
@@ -71,13 +88,16 @@ InitPaletteWithTable(
     _In_ PULONG Table,
     _In_ ULONG Count);
 
-/*
- * Globals
- */
-extern UCHAR VidpTextColor;
-extern ULONG VidpCurrentX;
-extern ULONG VidpCurrentY;
-extern ULONG VidpScrollRegion[4];
-extern UCHAR FontData[256 * BOOTCHAR_HEIGHT];
+VOID
+NTAPI
+VidpScroll(
```

Comment:
It will be much better if you also add the `Vidp` prefix to all functions exported from this shared header.

Reply:
Done.